hedera-node/yahcli/src/main/java/com/hedera/services/yahcli/commands/accounts/StakeCommand.java

```diff
@@ -50,9 +50,10 @@ public Integer call() throws Exception {
         final String target;
         final StakeSuite.TargetType type;
         final var normalizedElectedAccountNum = normalizePossibleIdLiteral(config, electedAccountNum);
-        if (normalizedElectedAccountNum != null) {
+        final var normalizedElectedNodeId = normalizePossibleIdLiteral(config, electedNodeId);
+        if (normalizedElectedNodeId != null) {
```

Comment:
Curious, in what case(s) is it important to check the elected node ID _before_ the elected account number? 

This tweak might be ok, but we have to be careful about changing this code since devops automation relies on it (even if it is a bug).

Reply:
Ahh I see, good catch. This change should be ok because this was a bug introduced recently